website/source/docs/telemetry/overview.html.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: "docs"
+page_title: "Overview"
+sidebar_current: "docs-telemetry-overview"
+description: |-
+  Overview of runtime metrics available in Nomad along with monitoring and
+  alerting.
+---
+
+# Telemetry Overview
+
+The Nomad client and server agents collect a wide range of runtime metrics
+related to the performance of the system. On the server side, leaders and
+followers have metrics in common as well as metrics that are specific to their
+roles. Clients have separate metrics for the host metrics and for
+allocations/tasks, both of which have to be [explicitly
+enabled][telemetry-stanza]. There are also runtime metrics that are common to
+all servers and clients.
+
+Metrics are aggregated on a ten second interval and are retained for one minute.
+[Gauges, counters and timers][metric-types] are supported.
+
+There are three ways to obtain metrics from Nomad:
+
+* Query the [/metrics API endpoint][metrics-api-endpoint] to return metrics for
+  the current Nomad process (as of Nomad 0.7). This endpoint supports Prometheus
+  formatted metrics.
+* Send the USR1 signal to the Nomad process. This will dump the current
+  telemetry information to STDERR (on Linux).
+* Configure Nomad to automatically forward metrics to a third-party provider.
+
+Nomad 0.7 added support for [tagged metrics][tagged-metrics], improving the
+integrations with [DataDog][datadog-telem] and [Prometheus][prometheus-telem].
+Metrics can also be forwarded to [Statsite][statsite-telem],
+[StatsD][statsd-telem] and [Circonus][circonus-telem].
+
+## Alerting
+
+The recommended practice for alerting is to leverage the alerting capabilities
+of your monitoring provider. Nomad’s intention is to surface metrics that enable
+users to configure the necessary alerts using their existing monitoring systems
+as a scaffold, rather than to natively support alerting. Here are a few common
+patterns:
+
+* Export metrics from Nomad to Prometheus using the [StatsD
+  exporter][statsd-exporter], define [alerting rules][alerting-rules] in
+  Prometheus, and use [Alertmanager][alertmanager] for summarization and
+  routing/notifications (to PagerDuty, Slack etc..). A similar workflow is
+  supported for [Datadog][datadog-alerting].
+
+* Periodically submit test jobs into Nomad to determine if your application
+  deployment pipeline is working end-to-end. This pattern is well-suited to
+  batch processing workloads.
+
+* Deploy Nagios on Nomad. Centrally manage Nomad job files and add the Nagios
+  monitor when a new Nomad job is added. When a job is removed, remove the
```

Comment:
do we have a link to this?

Reply:
I don't actually have a link or reference to this.